applications/nrf_desktop/doc/hids.rst

```diff
@@ -97,7 +99,7 @@ HID notifications
 =================
 
 The ``hid_notification_event`` is used to synchronize the information about enabling or disabling the HID notifications for the HID input report.
-The event is submitted when the |GATT_HID| calls a callback related to enabling or disabling the notifications and the event is received only by the ``hids`` application module.
+The event is submitted when the |GATT_HID| calls a callback related to enabling or disabling the notifications and the event is received only by the ``hids`` (Question: where does this "hids" refer to?) application module.
```

Comment:
`hids` here refers to `nrf_desktop_hids` (`HID Service module`) - an nRF Desktop application module